Assets/VRTK/Prefabs/Resources/Scripts/VRTK_PointerDirectionIndicator.cs

```diff
@@ -72,8 +83,30 @@ public virtual Quaternion GetRotation()
             return Quaternion.Euler(0f, transform.localEulerAngles.y + offset, 0f);
         }
 
+        /// <summary>
+        /// The SetMaterialColor method sets the current material colour on the direction indicator.
+        /// </summary>
+        /// <param name="color">The colour to update the direction indicatormaterial to.</param>
+        /// <param name="validity">Determines if the colour being set is based from a valid location or invalid location.</param>
+        public virtual void SetMaterialColor(Color color, bool validity)
+        {
+            validLocation.SetActive(validity);
+            invalidLocation.SetActive((displayOnInvalidLocation ? !validity : validity));
+
+            if (usePointerColor)
+            {
+                Renderer[] renderers = GetComponentsInChildren<Renderer>();
+                for (int i = 0; i < renderers.Length; i++)
+                {
+                    renderers[i].material.color = color;
+                }
+            }
+        }
+
         protected virtual void Awake()
         {
+            validLocation = transform.Find("ValidLocation").gameObject;
```

Comment:
Yes, I do! I've pulled it already to my project and everything is working as expected.